index.test.js

```diff
@@ -17,7 +17,7 @@ test('options can be customized', () => {
 		algorithm: 'SHA256',
 		period: 60,
 		digits: 8,
-		secret: base32.encode(Math.random().toString(16).slice(2)),
+		secret: base32.encode(Math.random().toString(16).slice(2)).toString(),
```

Comment:
This is put to a string because `base32.encode()` returns a `Buffer` but the official type signature is a `string`